### ISSUE TITLE
Public vs. Private

### ISSUE THREAD

**Issue by scepticalcat** [KarthickNarayana/toy_robot#8]
https://github.com/KarthickNarayana/toy_robot/blob/3d570a92d5c8b027e2456ec4f7bce47223719ecf/lib/toy_robot/robot.rb#L9

Similar to an earlier issue, there doesn't seem to be any thought put into the public vs. private interface meaning that methods that have no external relevance and shouldn't be accessible are still public.

**Comment by KarthickNarayana** [comment]
Yes  are pvt instance variable.   Relevant changes to test cases have been done.
